README.md

```diff
@@ -76,6 +76,7 @@ There are a few important considerations to be aware of:
 2. The job that executes the deployment must at minimum have the following permissions:
    - `pages: write`
    - `id-token: write`
+   - `actions: write`
```

Comment:
Is this correct? 🤔 

I would think the upload in `actions/upload-pages-artifact` would need `actions: write`, but that this one would only need `actions: read`. I am admittedly out of the loop, though.

Reply:
I think you're technically right? But in order to upload the artifact to artifacts v4 in the first place so that this action can download it again, it would need `actions: write`, so the technically correct example would probably lead to confusion.

Reply:
Some folks divide their deployment job/workflow run their build job/workflow, though, so accuracy is probably relevant. Maybe include a caveat? 🤷🏻 

```suggestion
   - `actions: read`
     - `actions: write` if you are also uploading the artifact in the same job
```

Reply:
Actually, just verified with the Actions Artifacts team that `actions: read` should be all that's required.

```suggestion
   - `actions: read`
```

Alternatively, we could update this to use some internal APIs (again) that would avoid the need for this extra permission altogether. 🤔